packages/accounts/plugingen/phases/plugin-actions/index.ts

```diff
@@ -43,15 +47,15 @@ export const PluginActionsGenPhase: Phase = async (input) => {
     .map((n) => {
       const argsParamString =
         n.inputs.length > 0
-          ? dedent`{ 
-                  args, 
-                  overrides, 
+          ? dedent`{
+                  args,
+                  overrides,
                   context,
                   account = client.account
               }`
-          : dedent`{ 
-                  overrides, 
-                  account = client.account 
+          : dedent`{
+                  overrides,
+                  account = client.account
```

Comment:
@moldy530 context is not needed here, wasn't here before either.

Reply:
it's a bug if it wasn't... see the above part of the ternary. that one includes `context`

Reply:
when there is no input, do we still need context?

Reply:
ah I see. I guess we dont have any plugin action methods with no input arg? Added context var here.